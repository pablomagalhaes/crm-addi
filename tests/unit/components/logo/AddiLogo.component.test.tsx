import React from 'react';
import { shallow } from 'enzyme';

import AddiLogo from '@/components/logo/AddiLogo.component';

describe('<AddiLogo />', () => {
  it('should match snapchat', () => {
    const wrapper = shallow(<AddiLogo />);
    expect(wrapper).toMatchSnapshot();
  });
});
