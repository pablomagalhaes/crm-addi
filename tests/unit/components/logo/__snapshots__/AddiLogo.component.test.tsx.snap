// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddiLogo /> should match snapchat 1`] = `
<svg
  height="32"
  viewBox="0 0 138 32"
  width="138"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M14.02,1.293c-.6,1.25-1.055,2.2-2.922,6.047-.465.949-1.055,2.18-1.32,2.734S7.2,15.4,4.648,20.688,0,30.324,0,30.359c0,.063,3.191,1.66,3.238,1.613.02-.02.773-1.566,1.684-3.445s3.492-7.23,5.75-11.891c6.012-12.453,7.25-15.016,7.25-15.059,0-.027-3.176-1.578-3.25-1.578C14.656,0,14.355.582,14.02,1.293Zm0,0"
    fill="#fff"
  />
  <path
    d="M20.758.773c-1.336.656-1.574.793-1.527.941.035.09,1.492,3.145,3.246,6.773s3.449,7.129,3.77,7.785c2.426,5.078,7.531,15.6,7.578,15.652a34.062,34.062,0,0,0,3.266-1.512c.016-.02-2.25-4.742-5.031-10.5S26.625,8.68,26.18,7.75C23.488,2.16,22.434,0,22.387,0,22.359,0,21.625.348,20.758.773Zm0,0"
    fill="#fff"
  />
  <path
    d="M43.758,15.953V31.637l1.891-.02,1.883-.027.027-15.664L47.578.273h-3.82Zm0,0"
    fill="#fff"
  />
  <path
    d="M50.215,2.043v1.77l4.895.043c4.5.039,4.957.055,5.758.23a12,12,0,0,1,6.27,3.3,11.078,11.078,0,0,1,3.3,8.391,12.2,12.2,0,0,1-.629,4.191,11.189,11.189,0,0,1-3.039,4.77,11.1,11.1,0,0,1-4.555,2.688c-1.758.566-2.129.594-7.289.641l-4.711.035v3.555l4.8-.043a51.629,51.629,0,0,0,5.746-.191,15.208,15.208,0,0,0,9.145-4.477,14.266,14.266,0,0,0,4.2-8.113,25.462,25.462,0,0,0,.082-5.52,14.119,14.119,0,0,0-4.812-9A16.425,16.425,0,0,0,59.813.363C59.266.32,56.875.273,54.52.273h-4.3Zm0,0"
    fill="#fff"
  />
  <path
    d="M83.691,15.953V31.637h3.82V.273h-3.82Zm0,0"
    fill="#fff"
  />
  <path
    d="M90.148,2.043V3.82l4.895.035c5.352.039,5.469.055,7.387.656a11.567,11.567,0,0,1,7.332,7.094,13.85,13.85,0,0,1-.406,9.453,11.763,11.763,0,0,1-6.316,6.055c-2.227.848-2.582.895-8.133.938l-4.758.047v3.539H94.09c5.82,0,7.3-.148,9.844-.977a15.141,15.141,0,0,0,10.352-14.887,14.212,14.212,0,0,0-4.3-10.852A14.22,14.22,0,0,0,106.023,2.1,14.847,14.847,0,0,0,101.941.684c-1.738-.348-2.9-.41-7.488-.41h-4.3Zm0,0"
    fill="#fff"
  />
  <path
    d="M123.645,15.926l.027,15.664,1.895.027,1.883.02V.273h-3.824Zm0,0"
    fill="#fff"
  />
  <path
    d="M20.777,20.867c-2.1.984-5.895,2.754-8.434,3.93s-4.6,2.188-4.613,2.242c0,.129,1.375,3.148,1.5,3.3.016.02,2.617-1.176,5.793-2.652S22,24.434,23.469,23.75l2.711-1.277c.047-.035-1.445-3.316-1.52-3.363C24.633,19.09,22.887,19.883,20.777,20.867Zm0,0"
    fill="#fff"
  />
  <path
    d="M134.844,26.348a2.659,2.659,0,0,0-1.437,4.359,2.473,2.473,0,0,0,1.965.863,2.618,2.618,0,0,0,2.445-3.6A2.7,2.7,0,0,0,134.844,26.348Zm1.637.621a2.282,2.282,0,0,1,.547,3.527,2.1,2.1,0,0,1-1.664.684,1.967,1.967,0,0,1-1.648-.7,2.314,2.314,0,0,1-.008-3.055A2.186,2.186,0,0,1,136.48,26.969Zm0,0"
    fill="#fff"
  />
  <path
    d="M134.453,28.891c0,1.223.008,1.27.18,1.223.148-.035.184-.129.184-.445,0-.375.02-.4.246-.4.172,0,.371.129.664.41.371.363.816.6.816.43a2.711,2.711,0,0,0-.371-.437l-.375-.387.273-.145a.8.8,0,0,0,.363-1c-.16-.371-.516-.508-1.3-.508h-.68Zm0,0"
    fill="#fff"
  />
</svg>
`;
